lib/ext/heroku/command/addons.rb

```diff
@@ -56,30 +56,54 @@ def index
     # addons:plans
     #
     # list all available add-on plans
+    def services
+      addon_services = api.request(
+        :expects  => [200, 206],
+        :headers  => { "Accept" => "application/vnd.heroku+json; version=3" },
+        :method   => :get,
+        :path     => "/addon-services"
+      ).body
+
+      display_table(addon_services, %w[name human_name], %w[Slug Name])
+      display "\nSee plans with `heroku addons:plans SERVICE`"
+    end
+
+    alias_command "addons:list", "addons:services"
+
+    # addons:plans SERVICE
     #
-    # --region REGION      # specify a region for add-on plan availability
-    #
-    #Example:
-    #
-    # $ heroku addons:plans --region eu
-    # === available
```

Comment:
The concept of `state` in this manner is really a service-level attribute. If heroku/api#3724 gets merged, I'll add it to the `addons:services` table.